heureka/ui/src/components/shared/Helper.jsx

```diff
@@ -10,3 +10,31 @@ export const listOfCommaSeparatedObjs = (objs, prop) => {
     .map((obj) => obj?.node?.[prop])
     .join(", ")
 }
+
+export const highestSeverity = (vulnerablities) => {
```

Comment:
Please don't forget to add tests for those helpers at some point :)

Reply:
done!